### PR TITLE
Run features in parallel sdk sdk

### PR DESCRIPTION
**Pull request by karanshah-browserstack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,127 @@
...
+                            cucumber.execution.parallel.config.fixed.parallelism = 5</configurationParameters>
+                    </properties>
+                    <argLine>
+                        -javaagent:${com.browserstack:browserstack-java-sdk:jar}
+                    </argLine>
+                </configuration>
+            </plugin>
+```
+* Install dependencies `mvn compile`
+* Run your test using mvn verify -P <your-profile-name>
```
Comment (kamal-kaur04) [inline README.md:118]: <your-profile-name> is not visible in README.md post rendering

Comment (kamal-kaur04) [reply README.md:118]: `<your-profile-name>`

File: browserstack.yml [hunk 2]
```diff
@@ -3,8 +3,8 @@
 # =============================
 # Add your BrowserStack userName and acccessKey here or set BROWSERSTACK_USERNAME and
 # BROWSERSTACK_ACCESS_KEY as env variables
-userName: BROWSERSTACK_USERNAME
-accessKey: BROWSERSTACK_ACCESS_KEY
+userName: YOUR_USERNAME
+accessKey: YOUR_ACCESS_KEY
```
Comment (kamal-kaur04) [inline browserstack.yml:8]: framework key ?

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,31 +1,127 @@
...
+@CucumberOptions(features = "src/test/resources/features/singlee.feature")
+```
+Replace the above code and use the Junit5 code as shown below.
+```
+@Suite
+@IncludeEngines("cucumber")
+@SelectClasspathResource("parallelFeatures")
+@ConfigurationParameter(key = GLUE_PROPERTY_NAME, value = "com.browserstack.cucumber")
+```
+* Make sure you replace your `junit-jupiter-engine` with `cucumber-junit-platform-engine` and add `cucumber-java`, `junit-platform-suite`, `junit-jupiter`  dependency in your pom.xml.
```
[the remark this answers is not in the document]
Comment (karanshah-browserstack) [reply README.md:48]: Mentioned below

**Review by kamal-kaur04 — APPROVED** [review]

**Pull request merged**